changelog.md

```diff
@@ -18,12 +18,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Component edition:
   * Allowed to have a predefined list of values for Array attribute.
 * Add new environment variable `KEEP_CYPRESS_ATTRIBUTE` to keep all cypress attribute in html.
+* Set the position of the component according to the position of the mouse when dropped on `#root`.
```

Comment:
```
* Component edition:
  * Allowed to have a predefined list of values for Array attribute.
* Modelization feature:
  * Allow to use the mouse coordinate when placing new component on drag and drop feature
* Add new environment variable `KEEP_CYPRESS_ATTRIBUTE` to keep all cypress attribute in html.
```

Or something like that, but put your "comment" in `* Modelization feature:` after `* Component edition:` block

Reply:
Done